admin/templates/configuration/etc/xrdssi.cnf

```diff
@@ -44,11 +44,11 @@ group_size = 10
 # Running the fast scheduler at high priority tends to make it use significant 
 # resources on a small number of queries.
 # priority_snail = -20
-# priority_slow = 2
-priority_slow = 4
+priority_slow = 2
+# priority_slow = 4
 # priority_med = 3
-# priority_fast = 4
-priority_fast = 2
+priority_fast = 4
+# priority_fast = 2
```

Comment:
OK, all of these values should be speciable via a config. Please don't use envars -- there are too many constants here.

Reply:
This change is already in the configuration file :-) ..unless I misinterpreted your comment.